core-api/src/main/java/com/optimizely/ab/config/audience/AudienceIdCondition.java

```diff
@@ -74,9 +74,9 @@ public Boolean evaluate(ProjectConfig config, Map<String, ?> attributes) {
             logger.error("Audience {} could not be found.", audienceId);
             return null;
         }
-        logger.debug("Starting to evaluate audience {} with conditions: \"{}\"", audience.getName(), audience.getConditions());
+        logger.debug("Starting to evaluate audience \"{}\" with conditions: {}.", audience.getId(), audience.getConditions());
```

Comment:
Let's get rid of this log message. I don't see this in other places.

Reply:
Actually this log is getting logged in all other sdks, including python (which we followed).